packages/bits/package.json

```diff
@@ -38,7 +38,7 @@
     "build-examples": "ng b demo",
     "prebuild-examples": "npm run compile-colors",
     "build-examples:prod": "npm run build-examples --prod",
-    "build-examples:watch": "npm run build-examples --watch",
+    "build-examples:watch": "ng b demo --watch",
```

Comment:
Do we need the same changes for `build-examples:prod` script?

Reply:
looks like not, it fails builds.